### PR TITLE
Vilma/issue/598

### LINKED ISSUE #598: fix: Role Descriptions
Update the way the role descriptions are shown on the recruitment page.

### PR DESCRIPTION
**Pull request by vilmaviola**
**What's new** 🤪
- Styling so it's more consistent over the whole page.
- Breaks at bullets in the role descriptions 

**Screenshots** 📷
Screenshot 2021-09-16 at 10 02 59

**Linked issue** 🤓
fix: Role Descriptions #598

### CODE REVIEW

File: src/components/Recruitment/index.js [hunk 1]
```diff
@@ -39,9 +39,12 @@ const Recruitment = () => {
                         }
                     }
                 </Sticky>
+                <div className='description-header'>
+                    <h2>Role Descriptions</h2>
```
Comment (linnea-bonnevier) [inline src/components/Recruitment/index.js]: Maybe we can change it to "Available roles" or "Open roles" just to make it clearer that those are the current open roles

**Review by linnea-bonnevier — CHANGES_REQUESTED** [review]
I think it looks great and the responsive screens are looking fantastic 😄
Just for clarification, I think we could change "role descriptions" to e.g. "available roles"

File: src/components/Recruitment/index.js (hunk 1 above)
Comment (vilmaviola) [reply src/components/Recruitment/index.js]: Okay yeah good idea! 😃

**Review by linnea-bonnevier — APPROVED** [review]
Amazing👏

**Review by sgranlund — APPROVED** [review]
I think it looks awesome! Great work Vilma

**Comment by linnea-bonnevier** [comment]
Merging so we can have it up before recruitment ends tomorrow✨

**Pull request merged**